boot/src/test/java/com.taobao.arthas.boot/DownloadUtilsTest.java

```diff
@@ -0,0 +1,40 @@
+package com.taobao.arthas.boot;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class DownloadUtilsTest {
+
+    @Test
+    public void testreadMavenReleaseVersion() {
+        Assert.assertNull(DownloadUtils.readMavenReleaseVersion(""));
+    }
+
+    @Test
+    public void testreadAllMavenVersion() {
+        Assert.assertEquals(new ArrayList<String>(), DownloadUtils.readAllMavenVersion(""));
+    }
+
+    @Test
+    public void testgetRepoUrl() {
+        Assert.assertEquals("http", DownloadUtils.getRepoUrl("https/", true));
+        Assert.assertEquals("https://maven.aliyun.com/repository/public", DownloadUtils.getRepoUrl("aliyun", false));
+    }
+
+    @Test
+    public void testreadMavenMetaData() {
+        Assert.assertNull(DownloadUtils.readMavenMetaData("", true));
+
+        Assert.assertNotNull(DownloadUtils.readMavenMetaData("center", false));
+    }
+
+    @Test
+    public void testdownArthasPackaging() throws IOException, SAXException, ParserConfigurationException {
+        DownloadUtils.downArthasPackaging("http://repo1.maven.org/maven2", true, "3.0.4", "savePath");
```

Comment:
What is tested here?

Reply:
That was more for the code coverage as it provided 39% of it. I wasn't sure if to keep it or not, but i will remove it.

Reply:
A test must always check what the method under test has done.